### PR TITLE
Expose paths for focusing replied-to timeline items by tapping on an in-reply-to message bubble

### PR DESCRIPTION
**Pull request by stefanceriu**

### CODE REVIEW

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift [hunk 1]
```diff
@@ -220,16 +220,22 @@ struct TimelineItemBubbledStylerView<Content: View>: View {
...
                     // the infinite width from increasing the overall width of the view.
-                    TimelineReplyView(placement: .timeline, timelineItemReplyDetails: replyDetails)
-                        .fixedSize(horizontal: false, vertical: true)
-                        .padding(4.0)
-                        .frame(maxWidth: .infinity, alignment: .leading)
-                        .background(Color.compound.bgCanvasDefault)
-                        .cornerRadius(8)
-                        .layoutPriority(TimelineBubbleLayout.Priority.visibleQuote)
+                    
+                    Button {
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift:228]: Did you check if this is exposed in voiceover? I have a feeling it'll be merged with the rest of the message so we might need to add an explicit accessibility action (or however we exposed the context menu on timeline items).

**Review by pixlwave — APPROVED** [review]
Very nice, thank you 🙏

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift [hunk 2]
```diff
@@ -220,16 +220,22 @@ struct TimelineItemBubbledStylerView<Content: View>: View {
...
+                        context.send(viewAction: .focusOnItem(itemID: timelineItem.id))
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift:229]: Waiiit a second. Shouldn't this focus on the replyDetails event ID 😉

Which I then think means we can't do a focusOnItem as we don't have enough information (the stable ID) to create a `TimelineItemIdentifier`.

**Review by pixlwave — CHANGES_REQUESTED** [review]
Just had a thought whilst working on the permalink side of things and came back to notice the below:

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemPlainStylerView.swift [hunk 3]
```diff
@@ -52,7 +52,12 @@ struct TimelineItemPlainStylerView<Content: View>: View {
                         Rectangle()
                             .foregroundColor(.compound.iconTertiary)
                             .frame(width: 4.0)
-                        TimelineReplyView(placement: .timeline, timelineItemReplyDetails: replyDetails)
+                        
+                        Button {
+                            context.send(viewAction: .focusOnItem(itemID: timelineItem.id))
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemPlainStylerView.swift:57]: (Same issue here too)

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift (hunk 1 above)
Comment (stefanceriu) [reply ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift]: Handled, it was overriding everything so we switched it to a tap gesture

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift (hunk 2 above)
Comment (stefanceriu) [reply ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift]: Of course 🤦, I took care of it 👍

File: ElementX/Sources/Screens/RoomScreen/RoomScreenInteractionHandler.swift [hunk 4]
```diff
@@ -230,8 +230,12 @@ class RoomScreenInteractionHandler {
                 }
             }
         case .reply:
+            guard let eventId = eventTimelineItem.id.eventID else {
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/RoomScreenInteractionHandler.swift:233]: > [Acronyms and initialisms](https://en.wikipedia.org/wiki/Acronym) that commonly appear as all upper case in American English should be uniformly up- or down-cased according to case conventions:

😉😉

```suggestion
            guard let eventID = eventTimelineItem.id.eventID else {
```

**Review by pixlwave — APPROVED** [review]
LGTM 👌

**Pull request merged**